src/DDTrace/Configuration.php

```diff
@@ -125,10 +125,18 @@ public function getGlobalTags()
      */
     public function appName($default = '')
     {
+        // Using the env `DD_SERVICE_NAME` for consistency with other tracers.
+        $appName = $this->stringValue('service.name');
+        if ($appName) {
+            return $appName;
+        }
+
+        // This is deprecated and will be removed in a future release
```

Comment:
Thoughts on logging a deprecation notice if this env var is set? :)

Reply:
If you agree with this approach, I would suggest to add a huge header in the release notes for users to be notified and in 2/3 release to issue a deprecation warning to reduce the risk that logs will be flooded with this error logs. What do you think?

Reply:
I was thinking of debug level log, but now that I think about it, no one will probably notice unless they are debugging on staging or locally. :) We could just focus on documenting it well here and in the docs if you think that's a good strategy.